### ISSUE TITLE
ascii UnicodeEncodeError on deploy with Dockerfile

### ISSUE THREAD

**Issue by pemcconnell** [deis/workflow#177]
I've ran into an issue with two containers that I've tried pushing today, which seem to be caused by the Dockerfile build process outputs (by npm)
```
Starting build... but first, coffee!
Starting pod dockerbuild-gradience-ec3a6229-e382687f
Pod spec: {
  "metadata": {
    "name": "dockerbuild-gradience-ec3a6229-e382687f",
    "namespace": "deis",
    "creationTimestamp": null,
    "labels": {
      "heritage": "dockerbuild-gradience-ec3a6229-e382687f"
    }
  },
  "spec": {
    "volumes": [
      {
        "name": "objectstorage-keyfile",
        "secret": {
          "secretName": "objectstorage-keyfile"
        }
      },
      {
        "name": "docker-socket",
        "hostPath": {
          "path": "/var/run/docker.sock"
        }
      }
    ],
    "containers": [
      {
        "name": "deis-dockerbuilder",
        "image": "quay.io/deis/dockerbuilder:v2.0.0-beta2",
        "env": [
          {
            "name": "DEBUG",
            "value": "1"
          },
          {
            "name": "TAR_PATH",
            "value": "home/gradience:git-ec3a6229/tar"
          },
          {
            "name": "IMG_NAME",
            "value": "gradience:git-ec3a6229"
          },
          {
            "name": "BUILDER_STORAGE",
            "value": "minio"
          }
        ],
        "resources": {},
        "volumeMounts": [
          {
            "name": "objectstorage-keyfile",
            "readOnly": true,
            "mountPath": "/var/run/secrets/deis/objectstore/creds"
          },
          {
            "name": "docker-socket",
            "mountPath": "/var/run/docker.sock"
          }
        ],
        "imagePullPolicy": "IfNotPresent"
      }
    ],
    "restartPolicy": "Never",
    "serviceAccountName": ""
  },
  "status": {}
}

download tar file complete
extracting tar file complete
Step 1 : FROM nodesource/wheezy:0.10.36
 ---> 47612a4e2bae
Step 2 : MAINTAINER Ryan Grieve <ryan@REMOVED.com>
 ---> Using cache
 ---> e145fa2133ef
Step 3 : RUN npm install --silent -g webpack
 ---> Running in 87f34df2e818
/usr/bin/webpack -> /usr/lib/node_modules/webpack/bin/webpack.js
Traceback (most recent call last):
  File "/deploy.py", line 102, in <module>
    log_output(stream, True)
  File "/deploy.py", line 21, in log_output
    print(stream_chunk.replace('\n', ''))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 44-46: ordinal not in range(128)
size of streamed logs 643
Waiting for the deis/dockerbuild-gradience-ec3a6229-e382687f pod to end. Checking every 100ms for 15m0s
Done
remote: 2016/04/20 16:05:06 Error running git receive hook [Build pod exited with code 1, stopping build.]
Checking for builder pod exit code
To ssh://git@deisREMOVED.com:2222/gradience.git
   70acd05..ec3a622  master -> master
```

The following Dockerfile is one which causes the problem:
```
FROM nodesource/wheezy:0.10.36
MAINTAINER Ryan Grieve <ryan@REMOVED.com>

RUN npm install -g webpack
RUN npm install -g webpack-dev-server

WORKDIR /app

EXPOSE 8080
```

I can get around this issue by removing the output from npm install in the Dockerfile, i.e.

```
FROM nodesource/wheezy:0.10.36
MAINTAINER Ryan Grieve <ryan@REMOVED.com>

RUN npm install -g webpack > /dev/null
RUN npm install -g webpack-dev-server > /dev/null

WORKDIR /app

EXPOSE 8080
```
which deploys just fine

**Comment by bacongobbler** [comment]
Looks like someone else has also hit this with npm: https://github.com/deis/dockerbuilder/issues/49

**Comment by helgi** [comment]
deis/dockerbuilder#49 has been closed now and the fix should have made it into Workflow 2.0. I'm going to close this ticket as fixed as well. Please reopen if you still experience this issue
